Sources/SPTPersistentCacheOptions.m

```diff
@@ -94,9 +94,9 @@ - (instancetype)initWithCachePath:(NSString *)cachePath
     
     _debugOutput = [debugCallback copy];
     
-    self.currentTimeSec = (currentTimeBlock ?
-                           currentTimeBlock :
-                           ^NSTimeInterval() { return [[NSDate date] timeIntervalSince1970]; });
+    self.currentTimeSec = currentTimeBlock ? currentTimeBlock : ^ NSTimeInterval() {
```

Comment:
Should probably be:
```objc
_currentTimeSec = [(currentTimeBlock ?: ^{
    return [[NSDate date] timeIntervalSince1970];
}) copy];
```

- Change it ivar access since we’re in the -init.
- Change to to `?:` so we don’t have to repeat the object we’re checking against `nil`.
- The compiler will infer the return type of the block, and would complain if we returned something else, so we can remove it.
- Make the copy explicit since we’re no longer using the property.